tests/ui/components/results/alternativeverticalscomponent.js

```diff
@@ -0,0 +1,28 @@
+import mockManager from '../../../setup/managermocker';
+import DOM from '../../../../src/ui/dom/dom';
+import StorageKeys from '../../../../src/core/storage/storagekeys';
+import AlternativeVerticalsComponent from '../../../../src/ui/components/results/alternativeverticalscomponent';
+
+DOM.setup(document, new DOMParser());
+let COMPONENT_MANAGER, defaultConfig;
+beforeEach(() => {
+  COMPONENT_MANAGER = mockManager();
+  COMPONENT_MANAGER.getComponentNamesForComponentTypes = () => [];
+  const bodyEl = DOM.query('body');
+  DOM.empty(bodyEl);
+  DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+
+  defaultConfig = {
+    container: '#test-component',
+    verticalsConfig: []
+  };
+});
+
+describe('AlternativeVerticals component', () => {
+  it('passes query to state correctly', () => {
```

Comment:
Could we test something a bit higher-level here? This test is verifying a lower-level implementation detail of the component: that query makes it into the state. These kinds of details can be subject to change. What's more isolated from change is the higher-level behavior of the component. Specifically, that the query appears in the alternative verticals text. Could we change the test to verify that the text is computed properly? It will have the side benefit of ensuring that the component somehow parses the query from globalStorage.

Reply:
I had a lot of trouble getting the alternativeVerticals component to render properly, so resorted to this, but can take another look!

Reply:
updated the test!